app/bundles/UserBundle/Translations/en_US/messages.ini

```diff
@@ -45,7 +45,7 @@ mautic.user.user.form.confirmbatchdelete="Delete the selected users?"
 mautic.user.user.form.confirmdelete="Delete the account of %name%?"
 mautic.user.user.form.defaultlocale="System Default Locale"
 mautic.user.user.form.defaulttimezone="System Default Timezone"
-mautic.user.user.form.help.passwordrequirements="Must contain at least 6 characters."
+mautic.user.user.form.help.passwordrequirements="Your password must be at least 6 characters and should use a combination of upper and lower case, special characters and numbers."
```

Comment:
Because we are using a third party library for the password strength, I am not sure this message is 100% correct. 

If a user has a password that meets these requirements will it always be strong enough to pass the validator? If not, it can lead to user frustration.

Maybe we can find a way to expose the info the password validator has instead of our own message?

Reply:
If you find a way how, then we can.

Reply:
Ill try to take a look today or this weekend